sqlglot/expressions.py

```diff
@@ -2603,10 +2603,24 @@ class DistKeyProperty(Property):
     arg_types = {"this": True}
 
 
+# https://docs.starrocks.io/docs/sql-reference/sql-statements/data-definition/CREATE_TABLE/#distribution_desc
+# https://doris.apache.org/docs/sql-manual/sql-statements/Data-Definition-Statements/Create/CREATE-TABLE?_highlight=create&_highlight=table#distribution_desc
+class DistributedByHashProperty(Property):
+    arg_types = {"expressions": True, "buckets": False, "order": False}
+
+
+class DistributedByRandomProperty(Property):
+    arg_types = {"expressions": False, "buckets": False, "order": False}
```

Comment:
I'd prefer merging these in a generic `class DistributedByProperty` using a `kind` keyword since they share similar args, position etc